src/main/scala/firrtl/passes/wiring/WiringUtils.scala

```diff
@@ -113,8 +115,64 @@ object WiringUtils {
     * @throws WiringException if a sink is equidistant to two sources
     */
   @deprecated("This method can lead to non-determinism in your compiler pass. Use sinksToSourcesSeq instead!", "Firrtl 1.4")
-  def sinksToSources(sinks: Seq[Named], source: String, i: InstanceGraph): Map[Seq[WDefInstance], Seq[WDefInstance]] =
-    sinksToSourcesSeq(sinks, source, i).toMap
+  def sinksToSources(sinks: Seq[Named], source: String, i: InstanceGraph): Map[Seq[WDefInstance], Seq[WDefInstance]] = {
```

Comment:
Is the reason you had to duplicate this function is because you needed the types of the WDefInstances?

Reply:
Yes. We can hopefully remove it as soon as 1.4 is released.